File: C++ Programming/problems on numberical/problem96.cpp
/*
96.Write a program which accept string from user and count number of capital characters. 
	
	Input :   “Marvellous Multi OS” 
	Output :   4

	Author : Milind Nalawade
	
	Date : 13/07/2021
	
	Language : C++ programming
	
*/
using namespace std;
#include<iostream>

int CountCapital(char* str)
{
	int no=0;
	
	while(*str!='\0')
	{
		if(*str>='A'&&*str<='Z')
		{	
			no++;
		}
		str++;
	}
	return no;
}

//Entry point function

int main()	
{
	char arr[20];
	int ret=0;
	cout<<"\n Enter your string : ";
	gets(arr);
	ret=CountCapital(arr);
	cout<<"\n The number of capital : "<<ret<<"\n";
	return 0;
}

File: Python Programming/problems on string manipulation/problem96.py
'''
97. Write a program which accept string from user and count number of captail characters. 

	Input :   "Marvellous"
	Output :   1
	Author : Milind Nalawade
	
	Date : 13/07/2021
	
	Language : Python 3.9.0	
'''
#====
#Logic
#====
def CountCaptial(str):
	no=0
	for i in str:
		if i>='A' and i<='Z':
			no+=1
	return no		
#=============
#Entry point function
#=============
def main():
	str=input("\n Enter your string : ")
	ret=int(CountCaptial(str))
	print("\n There are {} captial letters present in the string ".format(ret))
#=====
#starter
#=====
if __name__=="__main__":
	main()